addons/message_paid_research/manifest.json

```diff
@@ -5,7 +5,7 @@
   "type": "message",
   "translatable": false,
   "conditions": {
-    "min_client_version": "2.15.0",
+    "min_client_version": "100.15.0",
```

Comment:
Can you remove it entirely instead?

Reply:
yea, that probably makes the most sense if we are never going to re-use these